app/controllers/katello/api/v2/sync_plans_controller.rb

```diff
@@ -117,13 +117,6 @@ def sync
 
     protected
 
-    def find_plan
-      @sync_plan = SyncPlan.find_by(:id => params[:id])
-      fail HttpErrors::NotFound, _("Couldn't find sync plan '%{plan}' in organization '%{org}'") % { :plan => params[:id], :org => params[:organization_id] } if @sync_plan.nil?
-      @organization ||= @sync_plan.organization
```

Comment:
Did you account for this ? Is the `@organization` needed anywhere ?

Reply:
I dont think its needed. Just checking....

Reply:
I made a push to add this back although I am not sure if it's needed..I didn't see any issues without it.